src/ralph/data_center/tests/test_models.py

```diff
@@ -272,3 +272,47 @@ def test_change_rack_in_descendants(self):
     def test_get_autocomplete_queryset(self):
         queryset = DataCenterAsset.get_autocomplete_queryset()
         self.assertEquals(1, queryset.count())
+
+@ddt
+class RackTest(RalphTestCase):
+    def test_get_free_u_in_empty_rack_should_return_max_u_height(self):
+        rack = RackFactory()
+        self.assertEqual(rack.get_free_u(), rack.max_u_height)
+
+    @unpack
+    @data(
+        (1, 47, 47, 0),
+        (1, 46, 47, 1),
+        (2, 45, 47, 2),
+        (47, 1, 47, 46),
+        (47, 5, 47, 46),
```

Comment:
would you add comment for weirdos like me ;P
it could be
`# works ok even when position greater than max_u`